insights/client/new_refg.py

```diff
@@ -0,0 +1,90 @@
+def handle_registration(config):
```

Comment:
Is `new_refg.py` a typo?

Reply:
d'oh I meant to delete this file. It was a placeholder while I was changing stuff